app/views/users/registrations/edit.html.erb

```diff
@@ -28,29 +28,6 @@
     <%= f.file_field :photo %>
   </div>
 
```

Comment:
@shannonchu deleted this because it doesn't belong in edit-profile page. should make an edit account settings button for this